plugins/gfm/src/main/kotlin/GfmPlugin.kt

```diff
@@ -211,10 +211,14 @@ open class CommonmarkRenderer(
     }
 
     override fun StringBuilder.buildText(textNode: ContentText) {
-        val decorators = decorators(textNode.style)
-        append(decorators)
-        append(textNode.text)
-        append(decorators.reversed())
+        if(textNode.text.isNotBlank()) {
+            val decorators = decorators(textNode.style)
+            append(textNode.text.takeWhile { it == ' ' } )
```

Comment:
what is the reason we need to parse the text node while rendering it?

Reply:
There two bugs related to this:
- empty TextNodes with styles end up with e. g. "~~~~" representation, because it wants to surround empty text with styles
- there is some bug when styles characters (`*`, `**`, `~~`, etc) have white spaces at the beginning or at the end. If we provide text like `* there is some text *` vs ` *there is some text* `, the first one will end up rendered as it is with asterisks, the second one will be well formatted bold text (markserv tested)